lib/bundles/inspec-habitat/profile.rb

```diff
@@ -185,6 +187,12 @@ def create_run_hook
       File.write(run_hook_file, run_hook_contents)
     end
 
+    def create_settings_file
```

Comment:
Is this a habitat settings file? Are we potentially overwriting other habitat configs?